frontend/src/components/SavedSearch.jsx

```diff
@@ -32,7 +32,7 @@ export default function SavedSearches({ savedSearches, setSearchQuery,handleSave
                  className="dropdown-item"
                  onClick={() => handleSavedSearchClick(search.url)}
                >
-                 <p>{search.name}</p>
+                 <p>{search.name.replace(",+", " and ").replace(",", " and ")}</p>
```

Comment:
will need to be a `.replaceAll()` because right now this only corrects the first instance. 

If you search three different terms together only the first comma is corrected.

![Screenshot 2024-08-28 at 15 21 29](https://github.com/user-attachments/assets/1d1f3b5a-0178-4520-a20d-69669af2c191)